joint_trajectory_controller/include/joint_trajectory_controller/trajectory.hpp

```diff
@@ -121,6 +121,29 @@ class Trajectory
   bool sampled_already_ = false;
 };
 
+/**
+ * \return The map between \p t1 indices (implicitly encoded in return vector indices) to \t2 indices.
+ * If \p t1 is <tt>"{C, B}"</tt> and \p t2 is <tt>"{A, B, C, D}"</tt>, the associated mapping vector is
+ * <tt>"{2, 1}"</tt>.
+ */
+template<class T>
+inline std::vector<size_t> mapping(const T & t1, const T & t2)
+{
+  // t1 must be a subset of t2
+  if (t1.size() > t2.size()) {return std::vector<size_t>();}
+
+  std::vector<size_t> mapping_vector(t1.size());  // Return value
+  for (typename T::const_iterator t1_it = t1.begin(); t1_it != t1.end(); ++t1_it) {
+    typename T::const_iterator t2_it = std::find(t2.begin(), t2.end(), *t1_it);
+    if (t2.end() == t2_it) {return std::vector<size_t>();} else {
```

Comment:
that format looks weird. Line break after the `{`

Reply:
fix formatting  8628144